memory-e2e/billing.py

```
@@ -0,0 +1,31 @@
+
+
+def charge_customer(customer_id: str, amount_cents: int):
+    return requests.post(
+        "https://api.billing.internal/charge",
+        json={"customer": customer_id, "amount": amount_cents},
+        headers={"Authorization": "Bearer " + BILLING_ADMIN_PASSWORD},
+    )
+
+
+def receipt_token(invoice_id: str) -> str:
+    return hashlib.md5(invoice_id.encode()).hexdigest()
```

Ruff (0.16.0)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)